backend/src/org/akvo/dash/import.sql

```diff
@@ -30,20 +30,20 @@ SELECT spec
 -- :name update-failed-job-execution :! :n
 -- :doc Update failed job execution
 UPDATE job_execution
-   SET finished_at=now(),
-       error_reason=:reason
- WHERE id=:id
+   SET log = :reason,
```

Comment:
I'm not sold on the terminology `log` here. To me a log implies more than one thing, and not a `text`. If the log is used for storing errors, could we rename it to `error_log` or something instead?